src/assets/css/_colors.scss

```diff
@@ -0,0 +1,14 @@
+// Add new color properties here.
+// The color property variable should be in the form $color-<bg|font|icon|border|...>-<domain>[-subdomain]
+
+$color-bg-header: #b1daae;
+$color-bg-footer: #b1daae;
```

Comment:
Forslag: Gi variablene navn basert på fargen istedenfor hvor de brukes. Da slipper man å definere flere variabler for samme fargekode. De blir også mer generelle, som gjør det enklere å bruke de flere steder. Her kan du f.eks: `$color-green-light: #b1daae;`